### ISSUE TITLE
Task names with inline comment are not detected

### ISSUE THREAD

**Issue by liquidz** [kyu08/fzf-make#21]
## How to reproduce

```make
foo: # This is a comment
	echo "foo"

bar:
	echo "bar"
```

## Actual

* `fzf-make` only shows `bar`.

## Expected

* `fzf-make` will show `foo` and `bar`.

**Comment by kyu08** [comment]
Thank you for using this tool and taking the time to give me feedback. I feel very happy because this is the first time I get feedback on my project and know that it is being used by someone I don't know.

I will fix the problems you pointed out right away and release new version.

**Comment by kyu08** [comment]
@liquidz 
A bug-fixed version has been released as `v0.4.0`.
Please update fzf-make by executing the following command

```
brew update
brew upgrade fzf-make
```

Again, thank you for the report.

**Comment by liquidz** [comment]
@kyu08 It works good. Thanks for your quick fix!
